src/components/IconButton/__stories__/IconButton.stories.mdx

```diff
@@ -71,6 +91,8 @@ Please write here the component description
   </Story>
 </Canvas>
 
+<Tip>If you need to use an icon as a button that opens dialog or menu next to it, check out  <Link href="/?path=/docs/buttons-menubutton--overview" size={Link.sizes.SMALL} withoutSpacing>Menu button</Link> component</Tip>
```

Comment:
there is 2 link components in vibe - one is our system link and the other one is the storybook link (which support withoutSpacing prop because in some storybook components is needed) :)
the size and the withoutSpacing is depend on the container the story book link is present at.